src/index.js

```diff
@@ -0,0 +1,135 @@
+import './style.css';
+import './assets/img/square.svg';
+import './assets/img/check-square.svg';
+import './assets/img/more-vertical.svg';
+import './assets/img/bin.svg';
+import Lists from './module/listFuns.js';
+
+// add
+const todoEl = document.querySelector('.todo-list');
+const listFuns = new Lists();
+
+// remove
+const textnormal = (item) => {
+  item.parentElement.style.background = '#fff';
+  item.nextElementSibling.src = './assets/img/more-vertical.svg';
+  item.nextElementSibling.className = 'more';
+  item.disabled = true;
+};
+
+const removeTodo = () => {
+  todoEl.addEventListener('click', (e) => {
+    e.preventDefault();
+    const cur = e.target;
+    const curPrev = cur.previousElementSibling;
+    const curlist = cur.parentElement;
+    const curIdx = Array.from(todoEl.childNodes).indexOf(curlist);
+
+    if (cur.className.includes('more')) {
+      curlist.style.background = '#F8EDE3';
+      cur.src = './assets/img/bin.svg';
+      cur.className = 'bin';
+      if (curlist.firstElementChild.dataset.completed === 'false') {
+        curPrev.disabled = false;
+      } else {
+        curPrev.disabled = true;
+      }
+    } else if (cur.className.includes('bin')) {
+      listFuns.delList(curlist, curIdx);
+    }
+
+    if (curlist.lastElementChild.className === 'bin') {
+      if (cur.className === 'todo-text') {
+        curlist.addEventListener('focusout', () => {
+          textnormal(cur);
+        });
+      }
+    }
+  });
+};
+
+removeTodo();
+
+// edit
+const editTodo = () => {
+  const todoEl = document.querySelector('.todo-list');
+  const todoTexts = todoEl.querySelectorAll('.todo-text');
+  todoTexts.forEach((text, idx) => {
+    text.onchange = () => {
+      listFuns.edit(text.value, idx);
+    };
+
+    text.addEventListener('keydown', (e) => {
+      if (e.key === 'Enter' || e.key === 'Tab') textnormal(text);
+    });
+  });
+
+  localStorage.setItem('tdlists', JSON.stringify(listFuns.tdlists));
+};
```

Comment:
- [ ] It's clear that you add the `edit` function in `localstorage` as  it's part of the requirements (all changes are supposed to be stored in local storage), but the function it's not visible in live version. Maybe we can check that together just to be sure that is implemented!

Reply:
Thank You  .I've fixed it.